core/src/main/java/org/apache/druid/math/expr/Evals.java

```diff
@@ -20,21 +20,22 @@
 package org.apache.druid.math.expr;
 
 import org.apache.druid.common.config.NullHandling;
-import org.apache.druid.java.util.common.logger.Logger;
 
 import javax.annotation.Nullable;
-import java.util.Arrays;
 import java.util.List;
 
 /**
  */
 public class Evals
 {
-  private static final Logger log = new Logger(Evals.class);
-
   public static boolean isAllConstants(Expr... exprs)
   {
-    return isAllConstants(Arrays.asList(exprs));
+    for (Expr expr : exprs) {
```

Comment:
Why duplicate the code? To avoid wrapping the array?

Reply:
yeah, this change wasn't really related to anything, but it seemed a bit pointless to wrap it... though i guess probably not that expensive in the scheme of things since this is only used when flattening expressions in the `Parser`, so on second thought can just change it back